unraid_mcp/tools/health.py

```
@@ -1,186 +1,264 @@
+
+        # Resolve final status from severity level
+        severity_to_status = {v: k for k, v in _SEVERITY.items()}
+        health_info["status"] = severity_to_status.get(health_severity, "healthy")
+        if issues:
+            health_info["issues"] = issues
+        health_info["performance"] = {
+            "api_response_time_ms": api_latency,
+            "check_duration_ms": round((time.time() - start_time) * 1000, 2),
+        }
+
+        return health_info
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)